cve_bin_tool/checkers/firefox.py

```diff
@@ -17,7 +17,7 @@ class FirefoxChecker(Checker):
     CONTAINS_PATTERNS: list[str] = []
     FILENAME_PATTERNS: list[str] = [r"firefox"]
     VERSION_PATTERNS = [
-        r"ENCODER=Mozilla([0-9]+.[0-9]+(.[0-9]+)?)",
+        r"firefox([0-9]+.[0-9]+(.[0-9]+)?)",
```

Comment:
Is this a pattern existing in a Firefox binary?
The issue meant only the removal of the pattern to avoid false positives. But if you do find existing viable patterns feel free to add them.

Reply:
Oh sorry, I misunderstood the issue. I'll delete this PR and work on it.